main.go

```diff
@@ -70,6 +147,37 @@ func getSwap() int {
 	return int(v)
 }
 
+func getCredit() int {
+
+	instanceId, err := getInstanceId()
+
+	if err != nil {
+		return 0
+	}
+
+	commandStr := "aws cloudwatch get-metric-statistics --namespace AWS/EC2 --metric-name CPUCreditBalance --region ap-southeast-2 --dimensions Name=InstanceId,Value=" + instanceId + " --start-time $(date -d '10 minute ago' +%s) --end-time=$(date +%s) --period 3600 --statistics Minimum --unit Count | jq '.Datapoints[0].Minimum'"
+
+	var cmd *exec.Cmd
+	cmd = exec.Command("bash", "-c", commandStr)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return 0
+	}
+
+	v, err := strconv.ParseFloat(strings.TrimSpace(string(output)), 64)
+	if err != nil {
+		return 0
+	}
+
+	instance, err := getInstanceDetails()
+	if err != nil {
+		return 0
+	}
+	percentageUsage := (v / instance.MaximumCredits) * 100
+
+	return int(percentageUsage)
```

Comment:
Please condense the last two lines again, no need for a variable which just exists momentarily - even if its name does help explain what it does, so too does the function name.